packages/paste-core/components/tooltip/src/index.tsx

```diff
@@ -49,7 +49,7 @@ const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(({baseId, childre
   return (
     <>
       {React.Children.only(
-        <TooltipPrimitiveReference {...tooltip} ref={ref}>
+        <TooltipPrimitiveReference {...tooltip} ref={ref} {...children.props}>
```

Comment:
Does the tooltip primitive pass props to its children as well? If not I think this might end up causing some unexpected event handling. For example, if the child is an input, I think this would pass `onChange` to the tooltip primitive, instead of the input.

Reply:
It does both. `referenceProps` in the render function in the line below is both the tooltip and child props merged together. 

It makes sense, as the tooltip does need to know about what it's being placed on so it can keep track of it in the hook.

I does pass to the child because the tests pass. I could write another testing using a different type of child to prove there's no prop clash.